to.etc.domui/src/main/resources/to/etc/domui/util/messages.properties

```diff
@@ -98,6 +98,7 @@ v.bad.duration=Ongeldige duur (voorbeeld: 12d 8h)
 v.bad.percentage=Ongeldig percentage ({0})
 v.invalid.operator=Ongeldige zoek-operatie ({0}) in zoekveld
 v.invalid.not.unique=Waarde ({0}) al in gebruik
+v.invalid.regex=Falsches Regex-Muster: {0}, waarde ({1})
```

Comment:
If we need it, should be: Ongeldige regular expression: {0}, waarde {1}

Reply:
Ok, I saw that translation already in use in code in some sql tool... Ah I see, it was _de_ . Ok, I'll fix that to your proposed text.

Reply:
fixed translation